src/css/controls/states.less

```diff
@@ -11,8 +11,8 @@
         }
     }
 
-    .jw-controls {
-        background: @controlbar-background;
+    .jw-controls-backdrop {
+        background: linear-gradient(fade(@black, 40%), transparent) 0 0 / 100% 120px no-repeat transparent;
```

Comment:
@radium-v why aren't we using the controlbar-background ? What happens if we have a title, the cast icon, and the center play button ? The cast icon is displayed on the idle state when casting is enabled to allow users to start casting before playback begins. Cast icon appears at the bottom right, title at the top left, and center play button in the middle